timescaledb/quick-start/python.md

```diff
@@ -5,39 +5,31 @@ This quick start guide is designed to get the Python developer up
 and running with TimescaleDB as their database. In this tutorial,
 you'll learn how to:
 
-* [Connect to TimescaleDB](#new_database)
-* [Create a relational table](#create_table)
-* [Generate a Hypertable](/api/:currentVersion:/hypertable/create_hypertable)
-* [Insert a batch of rows into your Timescale database](#insert_rows)
-* [Execute a query on your Timescale database](#execute_query)
+* [Connect to TimescaleDB](#connect-python-to-timescaledb)
+* [Create a relational table](#create-a-relational-table)
+* [Create a hypertable](#create-hypertable)
+* [Insert a batch of rows into TimescaleDB](#insert-rows-into-timescaledb)
+* [Execute a query on TimescaleDB](#execute-a-query)
 
 ## Prerequisites
 
-To complete this tutorial, you will need a cursory knowledge of the Structured Query
-Language (SQL). The tutorial will walk you through each SQL command, but it will be
-helpful if you've seen SQL before.
+* At least cursory knowledge of Structured Query Language (SQL). The tutorial will walk you through each SQL command, 
```

Comment:
```suggestion
* At least some knowledge of SQL (structured query language). The tutorial will walk you through each SQL command, 
```

"some" just because it's a more commonly understood word. Switching the acronym/definition around because people are more likely to know "SQL" than the full spelled-out version.